modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -1577,7 +1577,7 @@ return view.extend({
 					o.rmempty = true;
 
 					o = ss.taboption('roaming', form.Flag, 'ft_psk_generate_local', _('Generate PMK locally'), _('When using a PSK, the PMK can be automatically generated. When enabled, the R0/R1 key options below are not applied. Disable this to use the R0 and R1 key options.'));
-					o.depends({ ieee80211r: '1' });
+					add_dependency_permutations(o, { ieee80211r: ['1'], mode: ['ap', 'ap-wds'], encryption: ['psk2', 'psk-mixed'] });
```

Comment:
Is the omission of SAE intentional?

Reply:
Yes, it must only be used for psk2 and psk/psk2 mixed